dependency-check-maven-config.xml

```diff
@@ -150,4 +150,21 @@
         <cve>CVE-2020-11980</cve>
     </suppress>
 
+    <suppress>
+        <notes>
+            Zookeeper is the only thing still using Log4J, and it has suppressions for both the
```

Comment:
Nice notes. We're using SLF4J, correct? 👍

Reply:
We mostly use the SLF4J api, though there's a few instances where we use others (Pax logging lets you use any of the standard logging apis). But the logging implementation we use is Log4J2